### ISSUE TITLE
download-object goal destinationdir does not work

### ISSUE THREAD

**Issue by robertpatrick** [kordamp/oci-maven-plugin#5]
I am trying to use the `download-object` goal but I am unable to specify the target directory to which to download the object.

The [guide](https://kordamp.org/oci-maven-plugin/#downloadObject) says that the goal supports a `destinationDir` parameter. 
 The Maven help:describe plugin says it is `destinationdir`.

```
% mvn help:describe -DgroupId=org.kordamp.maven -DartifactId=oci-maven-plugin -Dversion=0.5.0 -Dgoal=download-object -Ddetail
[INFO] Scanning for projects...
[INFO] 
[INFO] --------------------------< org.example:oci >---------------------------
[INFO] Building oci 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-help-plugin:3.3.0:describe (default-cli) @ oci ---
[INFO] Mojo: 'oci:download-object'
oci:download-object
  Description: (no description available)
  Implementation: org.kordamp.maven.plugin.oci.mojos.object.DownloadObjectMojo
  Language: java

  Available parameters:

    bucketName
      User property: oci.bucket.name
      (no description available)

    destinationdir
      User property: oci.destination.dir
      (no description available)

    fingerprint
      User property: oci.fingerprint
      (no description available)

    inputProperties
      User property: oci.input.properties
      (no description available)

    keyfile
      User property: oci.keyfile
      (no description available)

    namespaceName
      User property: oci.namespace.name
      (no description available)

    objectName
      User property: oci.object.name
      (no description available)

    passphrase
      User property: oci.passphrase
      (no description available)

    profile (Default: DEFAULT)
      User property: oci.profile
      (no description available)

    region
      User property: oci.region
      (no description available)

    showSecrets (Default: false)
      User property: oci.show.secrets
      (no description available)

    skip
      (no description available)

    tenantId
      User property: oci.tenantId
      (no description available)

    userId
      User property: oci.userId
      (no description available)


[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.938 s
[INFO] Finished at: 2022-10-24T09:40:59-05:00
[INFO] ------------------------------------------------------------------------
```

Unfortunately, neither one works.

```
[ERROR] Failed to execute goal org.kordamp.maven:oci-maven-plugin:0.5.0:download-object (get-file) on project oci: Unable to parse configuration of mojo org.kordamp.maven:oci-maven-plugin:0.5.0:download-object for parameter destinationdir: Cannot find 'destinationdir' in class org.kordamp.maven.plugin.oci.mojos.object.DownloadObjectMojo -> [Help 1]
```

**Comment by aalmiray** [comment]
Ah, perhaps I missed something. I can fix it an spin off `0.5.1` quickly. Is there something else that you'd like yo see included/fixed soon?

**Comment by robertpatrick** [comment]
@aalmiray see the other issue (#6) that I just filed.

**Comment by aalmiray** [comment]
Fixed in `0.5.1`
